packages/components/Menu/src/Menu/Menu.tsx

```diff
@@ -25,9 +26,25 @@ export const Menu = stagedComponent((props: MenuProps) => {
     const menuTrigger = childrenArray[0];
     const menuPopover = childrenArray[1];
 
+    const menuTriggerChild = menuTrigger.props.children;
+
+    const menuTriggerChildWithSet = React.cloneElement(
+      menuTriggerChild as React.ReactElement<unknown, string | React.JSXElementConstructor<any>>,
+      {
+        accessibilityPositionInSet: menuTriggerChild.props.accessibilityPositionInSet ?? accessibilityPositionInSet, // win32
```

Comment:
I wonder if it would be more robust to put these on the MenuContext instead... This would work when the MenuTrigger is the immediate child but I think we have clients who wrap the MenuTrigger in another component..